.babelrc

```diff
@@ -5,7 +5,7 @@
       {
         "targets": {
           "chrome": "104",
-          "node": 16
+          "node": "16.15.0"
```

Comment:
I don't understand why this change is necessary

Reply:
babel treats `"node": 16` as if it were `"node": "16.0.0"`. As there have been a lot of changes between node 16.0 and node 16.15.0 (the version used by the version of electron we are using), it makes sense to tell babel the exact version of node that we are targetting so that it can generate less pollyfills.

It isn't strictly necessary, though as their are only positive results of this change, it makes sense to do it.